lib/mix/lib/mix/cli.ex

```diff
@@ -126,4 +133,13 @@ defmodule Mix.CLI do
       ["--version", "-v"], do: :version
 
   defp check_for_shortcuts(_), do: nil
+
+  defp usage() do
```

Comment:
I have always been bothered by the fact that mix behaves different with the `--help` attribute.
I think we should make `mix --help` run the `mix help help` command. What do you think?